src/btree/bt_split.c

```diff
@@ -1339,7 +1319,7 @@ __wt_split_insert(WT_SESSION_IMPL *session, WT_REF *ref, int *splitp)
 	 */
 	page = NULL;
 	if ((ret = __split_parent(session, ref, split_ref, 2,
-	    parent_decr, parent_incr, 0, 0, &split_gen)) != 0) {
+	    parent_decr, parent_incr, 0, &split_gen)) != 0) {
 		/*
```

Comment:
I tried to understand that as well, but I couldn't see any reason to not cleanup overflow items: we're making a copy of the key...